### ISSUE TITLE
Feature request: use project 

### ISSUE THREAD

**Issue by AndreTheHunter** [ExplodingCabbage/sublime-gitignorer#14]
If in a project, please update the .sublime-project settings file instead of adding more to the global preferences file

**Comment by Narretz** [comment]
I agree. The readme doesn't make this clear either. I really didn't want a polluted preferences file.

**Comment by boneyfantaseas** [comment]
Me too. Also changing the project leaves me with gitignore setting which don't belong to my current project.
